rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala

```diff
@@ -900,4 +904,84 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
         } )
     } ) apply xml
   }
+
+  def nodeOnAcceptDefaultsConfiguration = { xml : NodeSeq =>
+
+    val modes = SelectableOption[Option[PolicyMode]](None, "Use global value") :: PolicyMode.allModes.map { x =>
+      SelectableOption[Option[PolicyMode]](Some(x), x.name.capitalize)
+    }.toList
+    // node states, sorted [init, enable, other]
+    val states = NodeState.values.toList.map { x => x match {
+      case NodeState.Initializing  => (0, x, x.name.capitalize)
+      case NodeState.Enabled       => (1, x, x.name.capitalize)
+      case NodeState.EmptyPolicies => (2, x, "Empty policies")
+      case NodeState.Ignored       => (3, x, x.name.capitalize)
+      case NodeState.PreparingEOL  => (4, x, "Preparing End Of Life")
```

Comment:
we have properties for these, we should use them

Reply:
(properties as we S.?

Reply:
I'm not sure I understand these two comments. Here, we are building the select dropdown, sorting values in a meaningful order (~the one of life cycle), and with human readable names.

Reply:
See https://github.com/Normation/rudder/blob/master/rudder-web/src/main/resources/default.properties#L95
we should use only one source for the labels, otherwise it will grow out of sync over the app